app/scss/components/_grid.scss

```diff
@@ -166,7 +166,14 @@
   @include flex(0, 0, 90%);
   max-width: 90%;
 }
-
+.left {
```

Comment:
I would rename these two clases into something different. And you might not need them at all. You can use col-sm-4 and col-sm-8. Please have a look on bootstrap grid